### PR TITLE
Gnw adjustments

### PR DESCRIPTION
**Pull request by SkewedAskew**
Small changes and fixes to some directional silliness

### Jab 1:
 - [$] Fixed the air puff appearing sooner than it should

### Jab Finisher:
 - [+] Hitbox Duration: F2 -> F2-4
 - [+] FAF: 28 -> 27

### Forward Tilt:
 - Late
   - [+] Damage: 6.0% -> 8.0%
   - [=] BKB: 35 -> 40
   - [=] KBG: 135 -> 108

### Up Tilt:
 - (*) Fixed the move having two more FAF when facing right
 - [+] FAF: 27/25 -> 25

### Forward Smash:
 - [/] Can now reverse hit
 - [-] Damage (late): 14.0% -> 12.0%
 - [-] FAF: 40 -> 43

### Up Smash:
 - (*) Fixed the move having vanilla data when facing right
 - [+] Damage: 16.0% -> 18.0%
 - [=] BKB: 40 -> 42
 - [-] KBG: 114/104 -> 100

### Neutral Air:
 - [+] Hitbox Duration (hit 4): F22-23 -> F22-24

### Judge:
 - Judge 1
   - [+] Matched aerial version to grounded version
 - Judge 6
   - [+] Damage: 12.0% -> 15.0%
   - [=] BKB: 30 -> 32
   - [=] KBG: 80 -> 66
 - Judge 9
   - [+] SDI Multiplier: 2.0x -> 1.0x

### Fire:
 - Hit 1
   - [/] Changed to a single hitbox
   - [-] Length decreased by 1.5u on each side
   - [-] Hitbox Size: 6.0u -> 5.0u

### Back Throw:
 - [+] Angle: 60 -> 68 (was supposed to match fthrow)

Screenshot 2023-07-19 18-42-00
Screenshot 2023-07-19 18-42-54

### CODE REVIEW

File: fighters/gamewatch/src/acmd/smashes.rs [hunk 1]
```diff
@@ -75,11 +44,11 @@ unsafe fn gamewatch_attack_hi4_game(fighter: &mut L2CAgentBase) {
...
     }
     frame(lua_state, 22.0);
     if is_excute(fighter) {
-        ATTACK(fighter, 0, 0, Hash40::new("top"), 16.0, 83, 104, 0, 40, 5.5, 0.0, 9.5, -6.0, Some(0.0), Some(8.0), Some(5.5), 1.0, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_PUNCH, *ATTACK_REGION_HEAD);
+        ATTACK(fighter, 0, 0, Hash40::new("top"), 18.0, 83, 100, 0, 42, 5.5, 0.0, 9.5, -6.0, Some(0.0), Some(8.0), Some(5.5), 1.0, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_L, *COLLISION_SOUND_ATTR_PUNCH, *ATTACK_REGION_HEAD);
```
Comment (SuddyN) [inline fighters/gamewatch/src/acmd/smashes.rs:51]: listed `[-] KBG: 114 -> 100` but is actually `[-] KBG: 104 -> 100`

File: fighters/gamewatch/src/acmd/specials.rs [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 use super::*;
 
 #[acmd_script( agent = "gamewatch", scripts = ["game_specialn", "game_specialairn"], category = ACMD_GAME, low_priority)]
 unsafe fn gamewatch_special_n_game(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = fighter.boma();
+    frame(lua_state, 1.0);
+    FT_MOTION_RATE_RANGE(fighter, 1.0, 18.0, 9.0);
     frame(lua_state, 18.0);
+    FT_MOTION_RATE(fighter, 1.0);
```
Comment (SuddyN) [inline fighters/gamewatch/src/acmd/specials.rs:10]: not listed in changelog?

**Review by SuddyN — DISMISSED** [review]

File: fighters/gamewatch/src/acmd/specials.rs (hunk 2 above)
Comment (SkewedAskew) [reply fighters/gamewatch/src/acmd/specials.rs:10]: This was motion rated in opff previously, this change brings it into acmd and cleans it up

File: fighters/gamewatch/src/acmd/smashes.rs (hunk 1 above)
Comment (SkewedAskew) [reply fighters/gamewatch/src/acmd/smashes.rs:51]: Vanilla kb

**Review by SuddyN — APPROVED** [review]
oops missed that necessary fighter param haha

**Pull request merged**